src/frontend/utils/formatter.ts

```diff
@@ -30,6 +30,8 @@ export const formaterIsoMånedÅr = (dato: string): string => {
     return parseISO(dato).toLocaleDateString('no-NO', datoMånedÅrFormat);
 };
 
+export const formaterMånedÅr = (dato: Date): string => format(dato, 'MM.yyyy');
```

Comment:
Denne skulle ikke brukes allikevel? Isåfall kan den kanskje slettes?

`formaterIsoMånedÅr` gjør egentlig samme nytten?